tsconfig.json

```diff
@@ -9,7 +9,6 @@
 		"esModuleInterop": true,
 		"module": "commonjs",
 		"moduleResolution": "node",
-		"importsNotUsedAsValues": "error",
```

Comment:
Seems a bit out of scope and something that shouldn't be removed.

Reply:
Should be done in whichever PR updates typescript to ^5.0, which happens to be this one right now.

Reply:
But this PR is formatting with prettier, and #6425 was closed without comment.